pkg/util/containers/filter.go

```diff
@@ -169,27 +169,25 @@ func GetSharedMetricFilter() (*Filter, error) {
 // GetPauseContainerFilter returns a filter only excluding pause containers
 func GetPauseContainerFilter() (*Filter, error) {
 	var excludeList []string
-	if config.Datadog.GetBool("exclude_pause_container") {
```

Comment:
why do we remove the `if`?

Reply:
This filter is now used to just identify whether a container is a pause container or not. We use this config value to determine whether to send the event to subscribers here: https://github.com/DataDog/datadog-agent/blob/08d815f1dc250d405bc70aee1862f299a6f6f21c/pkg/workloadmeta/filter.go#L62